cmd/zc_traverser_blob.go

```diff
@@ -95,15 +97,21 @@ func (t *blobTraverser) IsDirectory(isSource bool) bool {
 			msg := fmt.Sprintf("Failed to check if the destination is a folder or a file (Azure Files). Assuming the destination is a file: %s", err)
 			azcopyScanningLogger.Log(pipeline.LogError, msg)
 		}
-		return false
+		return false, nil
 	}
 
 	if len(resp.Segment.BlobItems) == 0 {
```

Comment:
Can we add a test for IsDirectory that tests a directory that only has virtual directories in it?

Example: Create a file called mydir/subdir1/file.txt and mydir/subdir2/file.txt and call isDirectory on mydir

Reply:
should fall under the NoStub test